### DISCUSSION TITLE
Error on github tab overview

### DISCUSSION THREAD

**Discussion by kalanakt** [community/community#31001] [Other Feature Feedback, Questions, & Ideas]
![bugreport#1](https://user-images.githubusercontent.com/86665964/187017785-ec325932-d89f-4819-9e5d-ea65f3fb93b7.png)

Year Buttons Not View Completly. when I scroll down it becomes normal

- operating system: windows 10
- browser: firefox

**Comment by radar-blip** [comment]
Did you try refreshing the page? Is it like this always?

**Reply by kalanakt** [reply]
Not Always. When I Open the Github profile in a new tab that will happen

**Reply by radar-blip** [reply]
Yes, it happened to me just a little while ago, same thing.
A glitch or bug I guess.

**Reply by kalanakt** [reply]
yup

**Reply by radar-blip** [reply]
Its on my profile page like that.
You can file a bug report if you like, here: [Bug-Report](https://support.github.com/contact/bug-report)

**Comment by jfullstackdev** [comment]
I also had the same experience but I think it's gone by now
